packages/office-addin-dev-settings/src/registry.ts

```diff
@@ -121,11 +121,13 @@ export async function deleteValue(key: RegistryKey, value: string): Promise<void
     };
 
     try {
-      key.winreg.valueExists(value, (valueExistsError, exists) => {
+      key.winreg.valueExists(value, (_, exists) => {
         if (exists) {
           key.winreg.remove(value, onError);
-        } else {
-          onError(valueExistsError);
+        }
```

Comment:
I should fix the formatting. I've been switching between different codebases that use different brace formatting styles.

Reply:
fixed